docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug:`2442` Users will now see their accounts sorted by name instead of address when they sort by account.
```

Comment:
Isn't this only on the assets view though? This should already be happening elsewhere.

Reply:
```suggestion
* :bug:`2442` Users will now see their accounts sorted by name instead of address when they sort by account in the assets view.
```